src/components/common/footer-button/button-spinner.js

```diff
@@ -0,0 +1,85 @@
+/** @jsx jsx */
+import { jsx, css, keyframes } from '@emotion/core'
+import React from 'react'
+
+const bounce = keyframes`
+  0%, 80%, 100% {
+              -webkit-transform: scale(0);
+              transform: scale(0);
+              }
+
+      40% {
+          -webkit-transform: scale(1);
+          transform: scale(1);
+          }
+
+        }
+`
+
+class ButtonSpinner extends React.Component {
```

Comment:
Why not use the spinner Patryk suggested?
https://codepen.io/camstephensdomo/pen/QYymQZ?page=1&

Reply:
No need to change tho it looks good too